tests/contrib/test_authentiq.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import unicode_literals
+
+import pytest
+import responses
+from flask import Flask
+from urlobject import URLObject
+
+from flask_dance.consumer import OAuth2ConsumerBlueprint
+from flask_dance.consumer.backend import MemoryBackend
+from flask_dance.contrib.authentiq import make_authentiq_blueprint, authentiq
+
+
+def test_blueprint_factory_default():
+    # Test with connect.authentiq.io
+    glbp1 = make_authentiq_blueprint(
```

Comment:
Can you name this variable `abp1` instead? (Short for Authentiq blueprint 1). You probably copied this from the GitLab automated tests, which is fine, but please make sure the variables reflect the content of the tests, here and throughout this file.

Reply:
Changed to `aqbp`, since that is the common abbreviation used.